src/fiatconnect/fiatAccountSchemas/pixAccount.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  EMAIL_REGEX,
+  FiatAccountSchema,
+  FiatAccountType,
+  PIXKeyTypeEnum,
+  PIX_CPF_KEY_REGEX,
+  PIX_PHONE_KEY_REGEX,
+} from '@fiatconnect/fiatconnect-types'
+import { getObfuscatedAccountNumber } from 'src/fiatconnect'
+import { INSTITUTION_NAME_FIELD } from 'src/fiatconnect/fiatAccountSchemas'
+import { FiatAccountFormSchema } from 'src/fiatconnect/fiatAccountSchemas/types'
+import i18n from 'src/i18n'
+
+const UUID_REGEX = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/g
+
+export const getPixAccountSchema = (): FiatAccountFormSchema<FiatAccountSchema.PIXAccount> => {
+  const keyValidation = (input: string, fieldNamesToValues: Record<string, string>) => {
+    const keyType = fieldNamesToValues['keyType'] as PIXKeyTypeEnum
+    let regex: RegExp
+    switch (keyType) {
+      case PIXKeyTypeEnum.EMAIL:
+        regex = EMAIL_REGEX
+        break
+      case PIXKeyTypeEnum.CPF:
+        regex = PIX_CPF_KEY_REGEX
+        break
+      case PIXKeyTypeEnum.PHONE:
+        regex = PIX_PHONE_KEY_REGEX
+        break
+      case PIXKeyTypeEnum.RANDOM:
+        regex = UUID_REGEX
+        break
+      default:
+        // Occurs when keyType isn't selected
+        regex = /.*/g
```

Comment:
if keyType isn't selected, key could still have invalid values-- anything that isn't a uuid, phone, cpf, or email. seems like a union of all the other regexes might be better for this